### PR TITLE
feat: Add new addresses (SC-788)

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: src/Base.sol [hunk 1]
```diff
@@ -7,14 +7,24 @@ library Base {
...
     /******************************************************************************************************************/
     /*** Bridging Addresses                                                                                         ***/
     /******************************************************************************************************************/
 
     address internal constant CCTP_TOKEN_MESSENGER = 0x1682Ae6375C4E4A97e4B583BC394c861A46D8962;
 
+    address internal constant L2_BRIDGE	      = 0xee44cdb68D618d58F75d9fe0818B640BD7B8A7B7;
+    address internal constant L2_BRIDGE_IMPL  = 0x289A37BE5D6CCeF7A8f2b90535B3BB6bD3905f72;
+    address internal constant L2_BRIDGE_SPELL = 0x6f29C3A29A3F056A71FB0714551C8D3547268D62;
+    address internal constant L2_GOV_RELAY    = 0xdD0BCc201C9E47c6F6eE68E4dB05b652Bb6aC255;
```
Comment (hexonaut) [inline src/Base.sol:26]: I would drop the `L2_` prefix and name these `TOKEN_BRIDGE` and `SKY_GOV_RELAY` respectively (drop the token bridge impl and bridge spell they are not important).

File: src/Ethereum.sol [hunk 2]
```diff
@@ -41,6 +41,25 @@ library Ethereum {
...
+    address internal constant ALLOCATOR_VAULT	 = 0x691a6c29e9e96dd897718305427Ad5D534db16BA;
+
+    /******************************************************************************************************************/
+    /*** SkyLink Bridge Addresses                                                                                   ***/
+    /******************************************************************************************************************/
+
+    address internal constant ESCROW	     = 0x7F311a4D48377030bD810395f4CCfC03bdbe9Ef3;
+    address internal constant L1_BRIDGE	     = 0xA5874756416Fa632257eEA380CAbd2E87cED352A;
+    address internal constant L1_BRIDGE_IMPL = 0xaeFd31c2e593Dc971f9Cb42cBbD5d4AD7F1970b6;
+    address internal constant L1_GOV_RELAY   = 0x1Ee0AE8A993F2f5abDB51EAF4AC2876202b65c3b;
```
Comment (hexonaut) [inline src/Ethereum.sol:61]: These are specific to base. I would refactor how we handle contracts related to particular L2s and put the name as the prefix: `BASE_ESCROW`, `BASE_TOKEN_BRIDGE` and `BASE_SKY_GOV_RELAY`. (Drop the bridge impl it's not really important)

I would group this into a section for Base that also includes the DSR/SSR oracle stuff below.

**Review by hexonaut — CHANGES_REQUESTED** [review]

File: src/Base.sol (hunk 1 above)
Comment (hexonaut) [reply src/Base.sol]: still need to drop impl

File: src/Ethereum.sol (hunk 2 above)
Comment (hexonaut) [reply src/Ethereum.sol]: same here to drop impl

File: src/Ethereum.sol [hunk 3]
```diff
@@ -182,13 +192,22 @@ library Ethereum {
...
     address internal constant DSR_FORWARDER_ARBITRUM    = 0x7F36E7F562Ee3f320644F6031e03E12a02B85799;
+    address internal constant DSR_FORWARDER_OPTIMISM    = 0x4042127DecC0cF7cc0966791abebf7F76294DeF3;
     address internal constant DSR_FORWARDER_WORLD_CHAIN = 0xA34437dAAE56A7CC6DC757048933D7777b3e547B;
 
+    /******************************************************************************************************************/
+    /*** Base Addresses                                                                                             ***/
+    /******************************************************************************************************************/
+
+    address internal constant DSR_FORWARDER_BASE = 0x8Ed551D485701fe489c215E13E42F6fc59563e0e;
     address internal constant SSR_FORWARDER_BASE = 0xB2833392527f41262eB0E3C7b47AFbe030ef188E;
```
Comment (hexonaut) [inline src/Ethereum.sol:204]: Consistent ordering: `BASE_` prefix instead of suffix.

**Review by hexonaut — CHANGES_REQUESTED** [review]
some comments still not addressed and let's drop all _IMPL

File: src/Base.sol (hunk 1 above)
Comment (lucas-manuel) [reply src/Base.sol]: Removed

**Review by hexonaut — APPROVED** [review]

**Review by hexonaut — APPROVED** [review]

**Pull request merged**